autoPyTorch/evaluation/tae.py

```diff
@@ -242,86 +355,81 @@ def run_wrapper(
                 starttime=time.time(),
                 endtime=time.time(),
             )
-        elif (
-                run_info.cutoff != int(np.ceil(run_info.cutoff))
-                and not isinstance(run_info.cutoff, int)
-        ):
-            run_info = run_info._replace(cutoff=int(np.ceil(run_info.cutoff)))
+        elif cutoff != int(np.ceil(cutoff)) and not isinstance(cutoff, int):
+            run_info = run_info._replace(cutoff=int(np.ceil(cutoff)))
 
-        self.logger.info("Starting to evaluate configuration %s" % run_info.config.config_id)
+        self.logger.info(f"Starting to evaluate configuration {run_info.config.config_id}")
         run_info, run_value = super().run_wrapper(run_info=run_info)
         return run_info, run_value
 
-    def run(
+    def _get_pynisher_func_wrapper_and_params(
         self,
         config: Configuration,
+        context: BaseContext,
+        num_run: int,
         instance: Optional[str] = None,
         cutoff: Optional[float] = None,
-        seed: int = 12345,
         budget: float = 0.0,
         instance_specific: Optional[str] = None,
-    ) -> Tuple[StatusType, float, float, Dict[str, Any]]:
+    ) -> Tuple[PynisherFunctionWrapperType, EvaluatorParams]:
 
-        context = multiprocessing.get_context(self.pynisher_context)
         preload_modules(context)
-        queue: multiprocessing.queues.Queue = context.Queue()
-
         if not (instance_specific is None or instance_specific == '0'):
             raise ValueError(instance_specific)
+
         init_params = {'instance': instance}
         if self.init_params is not None:
             init_params.update(self.init_params)
 
-        if self.logger_port is None:
-            logger: Union[logging.Logger, PicklableClientLogger] = logging.getLogger("pynisher")
-        else:
-            logger = get_named_client_logger(
-                name="pynisher",
-                port=self.logger_port,
-            )
-
         pynisher_arguments = dict(
-            logger=logger,
+            logger=_get_logger(self.fixed_pipeline_params.logger_port, 'pynisher'),
             # Pynisher expects seconds as a time indicator
             wall_time_in_s=int(cutoff) if cutoff is not None else None,
             mem_in_mb=self.memory_limit,
             capture_output=True,
             context=context,
         )
 
-        if isinstance(config, (int, str)):
-            num_run = self.initial_num_run
-        else:
-            num_run = config.config_id + self.initial_num_run
+        search_space_updates = self.fixed_pipeline_params.search_space_updates
+        self.logger.debug(f"Search space updates for {num_run}: {search_space_updates}")
```

Comment:
I dont think we need to specially create a variable just to log out the `search_space_updates`. I added it because I wanted to see if they are being passed correctly. especially when they will be printed for each run, when in fact they are the same for all num_runs.